pkg/services/store/entity/json.go

```diff
@@ -12,21 +12,12 @@ import (
 )
 
 func init() { //nolint:gochecknoinits
-	jsoniter.RegisterTypeEncoder("entity.EntitySearchResult", &searchResultCodec{})
```

Comment:
probably not something for this PR, but should new code move to another json package? it seems like [jsoniter is no longer maintained](https://github.com/json-iterator/go)

<img width="334" alt="image" src="https://github.com/grafana/grafana/assets/360020/d1f912c9-0695-4ac5-8615-633b48fdb108">

Reply:
I don't think we need any of the custom json stuff... it is all irrelevant now with apiserver